app/assets/stylesheets/places.scss

```diff
@@ -1,3 +1,34 @@
 // Place all the styles related to the places controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
```

Comment:
Probably it will be better to move that to some common file. This changes are not only related with places but with all app. This comment is just about name of the file.

Reply:
This still need to be fixed